Add installation docs for Rolling Ridley (#776)
ros2/ros2_documentation: source/Installation/Rolling/Linux-Install-Binary.rst

Installing ROS 2 on Linux
=========================

.. contents:: Table of Contents
   :depth: 2
   :local:

This page explains how to install ROS 2 on Linux from a pre-built binary package.

> As of Beta 2 there are also `Debian packages <Linux-Install-Debians>` available.

Reviewer: ```suggestion
There are also `Debian packages <Linux-Install-Debians>` available.
```

Reviewer 2: I've also opened https://github.com/ros2/ros2_documentation/pull/783 to make this change for other distros.

System Requirements
-------------------

We currently support Ubuntu Linux Focal Fossa (20.04) 64-bit x86 and 64-bit ARM.

> The Rolling Ridley distribution will change target platforms from time to time as new platforms become available.

Reviewer: ```suggestion
The Rolling Ridley distribution will change target platforms from time to time as new platforms are selected for development.
```
